src/containers/Tasks/SubmitTerms/SubmitTerms.tsx

```diff
@@ -166,8 +204,8 @@ class SubmitLegitimations extends React.Component<Props, State> {
         )}
 
         <>
-          <div className="selectLegitimations">
-            <h2>Select legitimation(s)…</h2>
+          <div className="selectTerms">
+            <h2>Select term(s)…</h2>
```

Comment:
I think that's the part that belongs to my comment/screenshot

Reply:
Updated.